### PR TITLE
Hw third part spring boot jpa

### PR DESCRIPTION
**Pull request by pikador10**

### CODE REVIEW

File: src/main/java/com/hw/bookstore/validation/BookValidation.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class BookValidation {
+
+    private final BookRepository bookRepository;
+
+    public void validateBookExistence(Long id) {
```
Comment (liliia-ponomarenko) [inline src/main/java/com/hw/bookstore/validation/BookValidation.java]: Do you really need a separate class for this method? 
Don't forget validate BookRequestDto using annotations ;)

File: src/main/resources/db/changelog/v1.0/changelog-v1.0.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+databaseChangeLog:
+  - include:
+      file: changes/2024_08_21_add_book_table.yaml
+      relativeToChangelogFile: true
```
Comment (liliia-ponomarenko) [inline src/main/resources/db/changelog/v1.0/changelog-v1.0.yaml:4]: why do you need this file? why not add the changeset files directly to the db.changelog-master.yaml?

File: src/main/resources/db/changelog/v1.0/changes/2024_08_21_add_book_table.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+databaseChangeLog:
+  - changeSet:
+      id: 2024_08_21_add_book_table
+      author: andrii
+      dbms: mysql,h2
+      changes:
+        - sqlFile:
+            dbms: mysql
+            encoding: utf8
+            path: ../data/2024-08-21_add_book_table.sql
```
Comment (liliia-ponomarenko) [inline src/main/resources/db/changelog/v1.0/changes/2024_08_21_add_book_table.yaml:10]: yaml script format is considered more flexible and recommended

**Review by liliia-ponomarenko — CHANGES_REQUESTED** [review]
Good job! Let's improve your solution;)

File: src/main/resources/db/changelog/v1.0/changelog-v1.0.yaml (hunk 2 above)
Comment (pikador10) [reply src/main/resources/db/changelog/v1.0/changelog-v1.0.yaml:4]: To me, it is a format that is easier to read. So keep it in life=)

File: src/main/resources/db/changelog/v1.0/changes/2024_08_21_add_book_table.yaml (hunk 3 above)
Comment (pikador10) [reply src/main/resources/db/changelog/v1.0/changes/2024_08_21_add_book_table.yaml:10]: I've used this format in my previous project and it was a wonderful experience. Try it.

File: src/main/java/com/hw/bookstore/validation/BookValidation.java (hunk 1 above)
Comment (pikador10) [reply src/main/java/com/hw/bookstore/validation/BookValidation.java]: This class has been removed.
Annotations will be added in the next PRs.

**Review by liliia-ponomarenko — APPROVED** [review]
Great! 😉

**Pull request merged**